wren-ui/src/apollo/server/dataSource.ts

```diff
@@ -151,6 +152,22 @@ const dataSource = {
     CLICK_HOUSE_CONNECTION_INFO,
     UrlBasedConnectionInfo
   >,
+  [DataSourceName.TRINO]: {
+    sensitiveProps: ['password'],
+    toIbisConnectionInfo(connectionInfo) {
+      const { catalog, host, password, port, schema, ssl, username } =
+        decryptConnectionInfo(
+          DataSourceName.TRINO,
+          connectionInfo,
+        ) as TRINO_CONNECTION_INFO;
+      console.log('connectionInfo', connectionInfo);
+      let connectionUrl = `jdbc:trino://${host}:${port}/${catalog}/${schema}?user=${username}&password=${password}`;
```

Comment:
According to the Wren Engine PR: https://github.com/Canner/wren-engine/pull/662
The connection Info in the URL format  should start with `tino://` instead of using the JDBC driver

I don't know if JDBC driver can be used in Python... 
I google it a little bit... and it seems like JDBC driver can be used in Python 3 [since 2015](https://pypi.org/project/JayDeBeApi/)

I've checked the [source code](https://github.com/ibis-project/ibis/blob/main/ibis/backends/trino/__init__.py#L28) of the ibis project and It just parses the URL and does not use the URL scheme specifier...so I guess it's fine